File: Swift/프로그래머스/2/12985. 예상 대진표/예상 대진표.swift
import Foundation

func solution(_ n:Int, _ a:Int, _ b:Int) -> Int
{
    var answer = 0
    var aCopy = a
    var bCopy = b

    while(aCopy != bCopy) {
        aCopy = (aCopy + 1)/2
        bCopy = (bCopy + 1)/2
        answer += 1
    }

    return answer
}

File: Java/프로그래머스/2/12985. 예상 대진표/예상 대진표.java
class Solution
{
    public int solution(int n, int a, int b)
    {
        int answer = 0;
        
        int aCopy = a;
        int bCopy = b;
        
        for (; aCopy != bCopy; answer++) {
            aCopy = (aCopy + 1) / 2;
            bCopy = (bCopy + 1) / 2;
        }

        return answer;
    }
}